docs/getting-started.md

```diff
@@ -120,12 +126,13 @@ Pushing image 172.30.1.1:5000/myproject/nodejs:latest ...
 Push successful
 ```
 
-Great news! Your component has been deployed on OpenShift now. Let's quickly check how it looks!
+Great news! Your component has been deployed to OpenShift! Now we'll connect to the component.
```

Comment:
nope ;-)
There is one missing step. You have to run `odo push` as `odo create --local` doesn't do push anymore.